libcudacxx/include/cuda/std/__ranges/owning_view.h

```diff
@@ -0,0 +1,161 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES
+//
+//===----------------------------------------------------------------------===//
+#ifndef _LIBCUDACXX___RANGES_OWNING_VIEW_H
+#define _LIBCUDACXX___RANGES_OWNING_VIEW_H
+
+#include <cuda/std/detail/__config>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include <cuda/std/__concepts/constructible.h>
+#include <cuda/std/__concepts/movable.h>
+#include <cuda/std/__ranges/access.h>
+#include <cuda/std/__ranges/concepts.h>
+#include <cuda/std/__ranges/data.h>
+#include <cuda/std/__ranges/empty.h>
+#include <cuda/std/__ranges/enable_borrowed_range.h>
+#include <cuda/std/__ranges/size.h>
+#include <cuda/std/__ranges/view_interface.h>
+#include <cuda/std/__type_traits/enable_if.h>
+#include <cuda/std/__type_traits/is_nothrow_default_constructible.h>
+#include <cuda/std/__type_traits/remove_cvref.h>
+#include <cuda/std/__utility/move.h>
+
+_LIBCUDACXX_BEGIN_NAMESPACE_RANGES
+
+_LIBCUDACXX_BEGIN_NAMESPACE_RANGES_ABI
+
+#if !defined(_CCCL_NO_CONCEPTS)
+template <range _Rp>
+  requires movable<_Rp> && (!__is_std_initializer_list<remove_cvref_t<_Rp>>)
```

Comment:
The [standard](https://eel.is/c++draft/range.owning.view) says:
```suggestion
  requires movable<_Rp> && (!__is_std_initializer_list<_Rp>)
```
Why is the `remove_cvref_t` needed?

Reply:
yeah, as always we would need to defer that to an impl and then in the proper one do the remove_cvref thingy